include/envoy/http/header_map.h

```diff
@@ -344,6 +343,7 @@ class HeaderEntry {
   HEADER_FUNC(Connection)                                                                          \
   HEADER_FUNC(ContentLength)                                                                       \
   HEADER_FUNC(ContentType)                                                                         \
+  HEADER_FUNC(EnvoyAttemptCount)                                                                   \
```

Comment:
It makes me happy to see this working properly. :)

Reply:
Yeah, this is awesome! Hard/tedious work is now paying off :) thanks for doing this. I took the liberty to narrow some more types in this PR to ensure static checking, really rad.